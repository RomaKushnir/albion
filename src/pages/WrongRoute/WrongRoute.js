import styles from './WrongRoute.module.scss';
import LinkButton from 'components/LinkButton';
import appRoutes from 'routes';

const WrongRoute = () => {
  return (
    <div className={styles.wrapper}>
      <h1>This page does not exist</h1>
      <LinkButton text="Go Home" path={appRoutes.ROOT} />
    </div>
  );
};

export default WrongRoute;
