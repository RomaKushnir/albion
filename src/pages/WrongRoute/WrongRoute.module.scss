.wrapper {
  display: flex;
  flex-direction: column;
  align-items: center;
  padding-top: 20vh;
  h1 {
    margin-bottom: 30px;
  }
}
