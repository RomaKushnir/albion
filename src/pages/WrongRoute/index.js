import WrongRoute from './WrongRoute';

export default WrongRoute;
