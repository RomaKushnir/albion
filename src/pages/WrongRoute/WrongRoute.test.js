import WrongRoute from '.';
import { shallow } from 'enzyme';

describe('render wrong route component', () => {
  const wrapper = shallow(<WrongRoute />);

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should have correct title', () => {
    expect(wrapper.find('h1').text()).toBe('This page does not exist');
  });
});
