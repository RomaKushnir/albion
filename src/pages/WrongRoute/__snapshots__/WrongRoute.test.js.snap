// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render wrong route component should render correctly 1`] = `
<div
  className="wrapper"
>
  <h1>
    This page does not exist
  </h1>
  <LinkButton
    path="/"
    text="Go Home"
  />
</div>
`;
